packages/excalidraw/components/App.tsx

```diff
@@ -4146,10 +4146,24 @@ class App extends React.Component<AppProps, AppState> {
           | ExcalidrawArrowElement
           | undefined;
 
-        const step = elbowArrow
-          ? elbowArrow.startBinding || elbowArrow.endBinding
-            ? 0
-            : ELEMENT_TRANSLATE_AMOUNT
+        const preventKeyboardMove = selectedElements
+          .filter(isElbowArrow)
+          .find((arrow) => {
+            const startMissing =
+              arrow.startBinding &&
```

Comment:
Why check for this if in the `find()` predicate `el.id` will never equal `undefined` if the `startBinding` itself is `undefined`.

Reply:
my mistake, was intended to be .some((arrow) => ...